zerver/tests/test_scim.py

```diff
@@ -186,6 +186,7 @@ def test_get_all_with_pagination(self) -> None:
         }
         for user_profile in UserProfile.objects.filter(realm=realm, is_bot=False).order_by("id"):
             user_schema = self.generate_user_schema(user_profile)
+            assert isinstance(expected_response_schema["Resources"], list)
```

Comment:
Mypy will infer this as `list[Any]`.

The mutation here is pointless; use a list comprehension instead.

```python3
        expected_response_schema = {
            "schemas": ["urn:ietf:params:scim:api:messages:2.0:ListResponse"],
            "totalResults": UserProfile.objects.filter(realm=realm, is_bot=False).count(),
            "itemsPerPage": 50,
            "startIndex": 1,
            "Resources": [
                self.generate_user_schema(user_profile)
                for user_profile in UserProfile.objects.filter(realm=realm, is_bot=False).order_by("id")
            ],
        }
```